### DISCUSSION TITLE
Hi we received this email, do you guys received too?

### DISCUSSION THREAD

**Discussion by yihong0618** [hozn/stravalib#449] [General]
![image](https://github.com/stravalib/stravalib/assets/15976103/c1ac786c-5281-4f05-9a7b-e0534e12539a)

also we had this discuss:
https://github.com/yihong0618/running_page/discussions/554

**Comment by jsamoocha** [comment]
Yes, all existing apps must apply to the new "Developer Program". My last PR #446 was also related to this.

**Reply by yihong0618** [reply]
thanks
